site/docs/parameter/parameter.md

```diff
@@ -37,10 +37,18 @@ Both variable and selection parameters support the following properties:
 
 ### Variable Parameters
 
-Variables are the simplest form a parameter can take. Variable parameters allow for a value to be defined once and then reused throughout the rest of the specification. As the name suggests, variable values can be modified dynamically either by [binding the variable](bind.html) to input widgets (e.g., sliders or drop down menus) or by specifying its `expr` property:
+Variables are the simplest form a parameter can take. Variable parameters allow for a value to be defined once and then reused throughout the rest of the specification. As the name suggests, variable values can be modified dynamically either by [binding the variable](bind.html) to input widgets (e.g., sliders or drop down menus), by specifying its `expr` property, or externally as a signal through the [Vega view API](https://vega.github.io/vega/docs/api/view/#signals):
```

Comment:
> externally as a signal through the [Vega view API]

This seems quite a leap. We should at least include an example?

Reply:
> by specifying its `expr` property

I think "modified" feels like a side effect, so binding and set via view  APIs feels like a match for "modified", but I think expr feels like more like declaring reactive dependencies.